lib/database_connection.rb

```diff
@@ -0,0 +1,11 @@
+require 'pg'
+
+class DatabaseConnection
```

Comment:
Module instead of class (optional). 
Classes are about objects; modules are about functions. 
Modules are about providing methods that you can use across multiple classes